UPGRADING.md

```diff
@@ -1,5 +1,15 @@
 # Upgrading ember-cli-flash
 
+## Upgrading to v6
```

Comment:
Assuming next release will be v6, although minor, this is a breaking change.

Would be good to batch all the remaining internal changes and TypeScript migration in the same update...